dash/dash.py

```diff
@@ -884,6 +894,7 @@ def add_context(*args, **kwargs):
                     mimetype='application/json'
                 )
 
+            self.callback_map[callback_id]['func'] = func
```

Comment:
Why do that in two difference part ? Couldn't your additions in dispatch be in `add_context` ?

Reply:
We would have to pass `namespace`, `component_id`, `component_property`, to this since that info is available in `dispatch`. I originally thought this might clash with the other arguments (e.g. if you had a property called `namespace`) although I now think this would be fine, but it might make it difficult if we want to support passing props with keyword arguments in the future.